### DISCUSSION TITLE
Can someone tell me  what Funky Mode is?

### DISCUSSION THREAD

**Discussion by neodelta22** [ebkr/r2modmanPlus#913] [Q&A]
its in the others section for settings, i currently have it disabled but have no idea what it does.

**Comment by woodstockfurygit** [comment]
Would really love an answer to this question. It's out there in the settings and it would be really beneficial for users to know exactly what this does...

**Comment by ToasterUwU** [comment]
@woodstockfurygit 

So i was wondering the same thing, but I'm a programmer and can read the code easily.

It doesn't work for me, but it should add [this image](https://github.com/ebkr/r2modmanPlus/blob/master/src/assets/funky_mode.png) over mod logos, change your font in the program to Comic Sans, and some over little goofy things.
